src/components/notifications/AppSelector/index.tsx

```diff
@@ -138,13 +138,15 @@ const AppSelector: React.FC = () => {
                     >
                       <div className="AppSelector__notifications">
                         <div className="AppSelector__notifications-link">
-                          <img
-                            className="AppSelector__link-logo"
-                            src={app.metadata.icons?.[0] || '/fallback.svg'}
-                            alt={`${app.metadata.name} logo`}
-                            onError={handleImageFallback}
-                            loading="lazy"
-                          />
+                          <div className="AppSelector__link-logo">
```

Comment:
I should have missed this while rebasing previous PRs, this has been made before. Fixing it